e2e/test/scenarios/filters-reproductions/dashboard-filters-reproductions.cy.spec.js

```diff
@@ -330,6 +331,31 @@ describe("issue 12720", () => {
     clickThrough("12720_SQL");
     clickThrough("Orders");
   });
+
+  it("should apply yhe specific (before|after) filter on a native question with field filter (metabase#47172)", () => {
+    visitDashboard(ORDERS_DASHBOARD_ID);
+
+    getDashboardCard(1).within(() => {
+      cy.findByTestId("TableFooter").should("exist");
+      cy.findByText("There was a problem displaying this chart.").should(
+        "not.exist",
+      );
+
+      cy.log("Drill down to the question");
+      cy.intercept("POST", "/api/card/*/query").as("cardQuery");
+      cy.findByText(questionDetails.name).click();
+    });
+
+    cy.location("search").should("eq", `?filter=${dashboardFilter.default}`);
+    cy.wait("@cardQuery").then(({ response }) => {
+      expect(response.body.status).to.eq("completed");
+    });
+    tableInteractive().should("be.visible").and("not.be.empty");
+    cy.findByTestId("question-row-count").should(
+      "not.have.text",
+      "Showing 0 rows",
+    );
```

Comment:
It's better to assert that the value is correct. 

```suggestion
    cy.findByTestId("question-row-count").should(
      "have.text",
      "Showing 1980 rows",
    );
```

We can have both assertions for extra clarity.


```suggestion
    cy.findByTestId("question-row-count").should(
      "not.have.text",
      "Showing 0 rows",
    );
    cy.findByTestId("question-row-count").should(
      "have.text",
      "Showing 1980 rows",
    );
```